### ISSUE TITLE
Request for a way to convert into a .dat file

### ISSUE THREAD

**Issue by EvilOcto1415** [KillzXGaming/Switch-Toolbox#566]
tl;dr I request a way to convert .bntx files to .dat files

So I'm currently working out converting a Bayonetta pc mod to work with the switch (or in this case yuzu, which I think is easier to mod). Bayonetta uses .dat files for basically everything. Pc Bayonetta also uses .dds for textures, but switch Bayonetta uses bntx. Those texture files are combined/compressed (really don't know how to describe it) into a .dat file. Unpacking the .dat I need into .dds files, and then converting those .dds files into .bntx files was the easy part. I just need a way to convert .bntx files into a .dat file (preferrably batch convert). Any way to implement that? If not I'll just stick with vanilla Bayonetta (I really wanteed to play kerminetta. Yes, Kermit the frog as Bayonetta. Very fun)
